code/modules/mob/living/simple_animal/hostile/mining_mobs/necropolis_tendril.dm

```diff
@@ -22,6 +22,7 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	maxbodytemp = INFINITY
+	anchored = TRUE
```

Comment:
this is inherited from `/mob/living/simple_animal/hostile/spawner` already, and thus does not fix shit all.